app/pages/Hive/DepositModal.tsx

```diff
@@ -86,11 +86,15 @@ export function DepositModal({ id, open, onClose, redirectTab, updateDeposit }:
 					showEffects: true,
 				});
 				logger.info({ msg: "Deposit tx:", method: "DepositModal", digest: result.digest });
+				// Force wait for the transaction to be fully indexed by the specific RPC node
+				await client.waitForTransaction({
+					digest: result.digest,
+				});
```

Comment:
you don't need to wait for the transaction like this here. Since you wait for effects, it will already wait for tx  execution